content/general/concepts/algorithm/terms/euclidean-algorithm/euclidean-algorithm.md

```diff
@@ -0,0 +1,149 @@
+---
+Title: 'Euclidean algorithm'
+Description: 'The Euclidean algorithm is a simple and efficient method for finding the highest common factor (HCF), also known as the greatest common divisor (GCD), of two numbers.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Algorithms'
+  - 'Hcf'
```

Comment:
```suggestion
  - 'Arithmetic'
```
Reminder, tags need to come from [this list](https://github.com/Codecademy/docs/blob/main/documentation/tags.md).

Reply:
Got it. We have to open a separate PR to add tags in tag.md?